packages/firebase_messaging/firebase_messaging/android/src/main/java/io/flutter/plugins/firebase/messaging/FlutterFirebaseMessagingBackgroundService.java

```diff
@@ -33,7 +32,8 @@ public static void enqueueMessageProcessing(Context context, Intent messageInten
         context,
         FlutterFirebaseMessagingBackgroundService.class,
         FlutterFirebaseMessagingUtils.JOB_ID,
-        messageIntent);
+        messageIntent,
+        true);
```

Comment:
With this we handle all messages in a wakeful service. Should we somehow check for message priority and only handle high priority data messages this way?